hack/update-vendor.sh

```diff
@@ -216,10 +242,6 @@ kube::util::list_staging_repos \
 
 # pin referenced versions
 ensure_require_replace_directives_for_all_dependencies
-# resolves/expands references in the root go.mod (if needed)
-go mod tidy >>"${LOG_FILE}" 2>&1
-# pin expanded versions
-ensure_require_replace_directives_for_all_dependencies
```

Comment:
remind me why we're dropping this bit? isn't dropping the expanding / pinning problematic?